app/src/main/java/eu/faircode/email/MessageHelper.java

```diff
@@ -624,13 +620,10 @@ else if (content instanceof InputStream)
             if (TextUtils.isEmpty(charset)) {
                 if (BuildConfig.DEBUG)
                     warnings.add(context.getString(R.string.title_no_charset, ct.toString()));
-                if (part.isMimeType("text/plain"))
-                    try {
-                        // The first 127 characters are the same as in US-ASCII
-                        result = new String(result.getBytes("ISO-8859-1"));
-                    } catch (UnsupportedEncodingException ex) {
-                        warnings.add(Helper.formatThrowable(ex));
```

Comment:
The warning should stay or is there a good reason to change this?

Reply:
Android Studio automatically removed it because `UnsupportedEncodingException` was thrown by `Charset.forNameUEE` which is called inside `String.bytes(String)`.

Since `String.bytes(String)` is replaced with a call to `String.bytes(Character)` in this commit the `UnsupportedEncodingException` isn't thrown anymore.